torch/onnx/_internal/fx/exporter.py

```diff
@@ -37,15 +36,17 @@ def _export(
     # Ensure placeholder targets match the original module's signature since
     # We don't want to map forward(x, y, z) to forward(arg0, arg1, arg2).
     decomposed_module = passes.decompose(
-        module, export_options.decomposition_table, *args
-    )
-    # Run FakeTensorProp on decomposed_module.
-    # Symbolic output of the i-th node can be accessed via
-    # decomposed_module.graph.nodes[i].meta["val"]
-    decomposed_module = passes.shape_inference_with_fake_tensor(
-        decomposed_module, *args
+        module, export_options.decomposition_table, export_options.dynamic_axes, *args
     )
-
+    # TODO(titaiwang): in fake mode we have seen fx.nodes without node.meta["val"]
+    # Do we need this in symbolic mode?
+    if not export_options.dynamic_axes:
```

Comment:
@BowenBao @wschin https://github.com/pytorch/pytorch/pull/96879#issuecomment-1477181945

Reply:
So only one of the `fake` mode test would lack of node.meta["val"]. I guess if all nodes should be expected to have node.meta["val"], we should look into model instead? (give up this pass)

Reply:
I killed the pass